packages/@aws-cdk/aws-apigatewayv2/lib/http/authorizer.ts

```diff
@@ -248,7 +248,7 @@ export interface IHttpRouteAuthorizer {
   bind(options: HttpRouteAuthorizerBindOptions): HttpRouteAuthorizerConfig;
 }
 
-function undefinedIfNoKeys<A>(obj: A): A | undefined {
+function undefinedIfNoKeys<A extends { [key: string]: unknown }>(obj: A): A | undefined {
```

Comment:
[breadcrumb] Anything is assignable to unknown, but unknown isn't assignable to anything but itself. Diff from `any`.